pingpong-go/src/pingpong/pong/Pong.go

```diff
@@ -1,29 +1,35 @@
-package pong 
+package pong
 
 import (
+	"bytes"
 	"fmt"
 	"net/http"
-	"github.com/gorilla/mux"
 	"strconv"
-	"bytes"
+
+	"github.com/gorilla/mux"
 )
 
 func pongHandler(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
-	i, _ := strconv.Atoi(params["length"])
+	i, err := strconv.Atoi(params["length"])
+	if err != nil {
+		panic(err)
+	}
 	i -= 4
-	var result bytes.Buffer
-	result.WriteString("po")
-	for ; i > 0; i-- { result.WriteString("o") }
+	result := bytes.NewBufferString("po")
```

Comment:
This might influence the benchmark negatively? What is faster bytes.Buffer or bytes.NewBufferString?
Without any reference on performance related publications regarding both approaches this pull request might slow down the benchmark ...